docker.go

```diff
@@ -164,9 +173,28 @@ func (p Plugin) Exec() error {
 		}
 	}
 
-	if p.Build.Squash && !p.Daemon.Experimental {
-		fmt.Println("Squash build flag is only available when Docker deamon is started with experimental flag. Ignoring...")
-		p.Build.Squash = false
+	// cache export feature is currently not supported for docker driver hence we have to create docker-container driver
+	if len(p.Build.CacheTo) > 0 && (p.Builder.Driver == "" || p.Builder.Driver == defaultDriver) {
```

Comment:
If we are using cache from, don't we need to set the driver?

Reply:
`docker` driver (default) supports cache from. It does not support cache to: https://docs.docker.com/engine/reference/commandline/buildx_build/

<img width="985" alt="image" src="https://user-images.githubusercontent.com/103484777/232418122-76b61fea-9b17-4364-8526-eaa73e2408ef.png">